negotiation/proposed.emu

```diff
@@ -359,13 +359,19 @@
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-getbooleanorstringnumberformatoption" aoid="GetBooleanOrStringNumberFormatOption">
-      <h1>GetBooleanOrStringNumberFormatOption ( _options_, _property_, _stringValues_, _fallback_ )</h1>
-
-      <p>
-        The abstract operation GetBooleanOrStringNumberFormatOption extracts the value of the property named _property_ from the provided _options_ object. It returns either *true*, *false*, _fallback_, or one of the elements of the List _values_. The following steps are taken:
-      </p>
-
+    <emu-clause id="sec-getbooleanorstringnumberformatoption" type="abstract operation">
+      <h1>
+        GetOption (
+          _options_: an Object,
+          _property_: a property key,
+          _stringValues_: ~empty~ or a List of ECMAScript String values,
+          _fallback_: an ECMAScript language value,
+        )
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It extracts the value of the property named _property_ from the provided _options_ object. It returns either *true*, *false*, _fallback_, or one of the elements of the List _stringValues_.</dd>
```

Comment:
This description can be improved along the lines of [GetOption](https://tc39.es/ecma402/#sec-getoption).
```suggestion
        <dd>It extracts the value of the property named _property_ from the provided _options_ object. It returns _fallback_ if that value is *undefined*, *true* if that value is *true*, *false* if that value coerces to *false*, and otherwise coerces it to a String and returns the result if it is allowed by _stringValues_.</dd>
```